tests/EloquentRepositoryTest.php

```diff
@@ -464,7 +464,8 @@ public function testItModifiesQueries()
         $this->assertEquals($repository->count(), 1);
         $repository->modify()->set([
             function (Builder $query) {
-                $query->whereRaw(DB::raw('0 = 1'));
+                //Force the query to return no results
+                $query->where('id', 'a-value-that-will-not-be-found');
```

Comment:
Laravel 10 no longer allows an implicit cast to string for `DB::raw(...)` expressions.
https://laravel.com/docs/10.x/upgrade#database-expressions

I updated this expression to avoid the use of `DB::raw(...)` so it can test across Laravel 8, 9, and 10 without a workaround.